### ISSUE TITLE
Map Case not working

### ISSUE THREAD

**Issue by gerard0315** [root-two/react-native-drawer#139]
Hi I tested the Map view case but seems not working. 
I'm using RN 0.24

**Comment by rt2zz** [comment]
What part of the mapview/drawer is not working?

**Comment by jeveloper** [comment]
I could comment on this, dragging  it from left to right and if you have a view that is taking the whole screen, the map will start moving along with the drawer.

**Comment by rt2zz** [comment]
Yes this is tricky with maps, I believe the map pan responder is native and does not respect the react-native pan responder negotiation (I may be wrong). The simplest solution I have seen is to mask the left side of your map with a transparent View.

**Comment by jeveloper** [comment]
good suggestion @rt2zz , thank you

have you seen an issue with callout expanded moving around when tapped on the pin (this is a custom view with an image on it) , the callout is also an image and appears whereever the person tapped on the pin and not at the center of the actual lat/long

**Comment by rt2zz** [comment]
never seen that. Are you using react-native-maps or the core MapView? I would highly recommend the former. https://gitter.im/lelandrichardson/react-native-maps

**Comment by jeveloper** [comment]
Yup using react-native-maps :(
